shenanigan/metrics/InceptionScore.py

```diff
@@ -0,0 +1,55 @@
+import os
```

Comment:
The file should be named with snake case

Reply:
@alecokas fixed